server/controllers/skills/addSkills.js

```diff
@@ -1,3 +1,16 @@
-module.exports = (req, res) => {
-  res.send(' add new skills ');
+const { checkSkills, insertSkills } = require('../../database/queries/skills/index');
+
+module.exports = (req, res, next) => {
+  const { name } = req.body;
+  checkSkills(name)
```

Comment:
```javascript
checkSkills(name)
    .then((result) => {
      if (result.rows.length !== 0) {
        return insertSkills(name)
            .then(response => res.send({
                    error: null,
                    data: response.rows[0],
              }))
          } 
          next({ code: 400, msg: 'skill already exist' });
    })
    .catch(err => next(err));
```

Reply:
@Fatmasiam 
```javascript
   if (result.rows.length !== 0) 
```
 ???? 
 we need to check if there isnt ```skill.name``` same with ```name``` from body req.